### PR TITLE
[WIP] add password reset

### LINKED ISSUE #434: Add a password reset

### PR DESCRIPTION
**Pull request by boney-bun**
fix #434

### CODE REVIEW

**Comment by boney-bun** [comment]
PIA:
ford3#436

**Comment by pierrealixt** [comment]
@boney-bun 
Ok the functionality is working fine but the UI is not good.
Few ideas to play with:
- the title OpenEdu in the box might be a bit too much
- the blue link 'Forget password' should not be blue, look at the mockup, it is white.
- the 'change my password' button is not styled
- the email input should have a placeholder
- something is definitively wrong with that big white footer

**Review by pierrealixt — COMMENTED** [review]
The functionality is working but the UI could be better:
- we don't need to repeat the OpenEdu title in the box.
- add a placeholder in the input text for the email
- same for new password input and confirmation
- that big white footer is not pretty

**Comment by boney-bun** [comment]
latest changes:
ford3#436_rev

**Comment by boney-bun** [comment]
the login page:
Screen Shot 2019-05-31 at 1 21 11 PM

the password reset page:
Screen Shot 2019-05-31 at 1 21 32 PM

**Comment by boney-bun** [comment]
set a password page:

Screen Shot 2019-05-31 at 1 34 42 PM

**Comment by pierrealixt** [comment]
what about the orange button Boney?

**Comment by pierrealixt** [comment]
@boney-bun 
According to your screenshot, the button is no more orange...
Please post a gif of the complete process. 

And please, please, before pushing your code, make sure `make flake8` runs fine.

**Comment by boney-bun** [comment]
9fa767d:
ford3#436

File: django_project/ford3/templates/registration/login.html [hunk 1]
```diff
@@ -37,10 +38,18 @@ <h4 class="text-center text-white">Login</h4>
               <form method="post" class='text-white'>
                 {% csrf_token %}
                 {{ form.as_p }}
+                <div class="row">
+                    <div class="col-md-12 py-2">
+                        <a href="{% url 'password_reset'%}" class="text-white">Forgot password</a>
```
Comment (pierrealixt) [inline django_project/ford3/templates/registration/login.html]: `Forgot password?`

**Review by pierrealixt — APPROVED** [review]
LGTM

**Pull request merged**